extensions/wikia/WallNotifications/WallNotificationEntity.class.php

```diff
@@ -78,174 +127,202 @@ public function getUniqueId() {
 		}
 	}
 
```

Comment:
Why have a single getter for ```id``` when none of the other members have getters and this one is not used uniformly?  I played with adding more getters but Wall code liberally accesses the ```data``` member and in ways that PHPStorm didn't recognize.  Eliminating all getters seemed a safer way to achieve consistency.